RocketApp/Domain/Sources/RocketsClient/RocketClient+liveValue.swift

```diff
@@ -11,9 +10,11 @@ extension RocketsClient: DependencyKey {
     return Self(
       getRocket: { id in
         Request(endpoint: URLs.baseURL + "/v3/rockets/\(id)").execute(using: networkClientType)
+          .ignoreFailure(setFailureType: RocketNetworkError.self)
```

Comment:
Tohle je blbost. Uplne zbytecne tim erasujes typ toho erroru. Zase je treba se naucit pouzivat `ErrorReporting` vrstveni a vhodne to pouzit.

Reply:
Napis si sem aspon TODO, ze to predelas pak na ten `ErrorReporting`.

Reply:
Pridano TODO.